spec/schemas/_core.msearch.yaml

```diff
@@ -119,44 +119,44 @@ components:
           $ref: '_core.search.yaml#/components/schemas/SourceConfig'
         fields:
           description: |-
-            Array of wildcard (*) patterns. The request returns values for field names
-            matching these patterns in the hits.fields property of the response.
+            An array of wildcard (*) patterns. The request returns values for field names
+            matching these patterns in the `hits.fields` property of the response.
           type: array
           items:
             $ref: '_common.query_dsl.yaml#/components/schemas/FieldAndFormat'
         terminate_after:
           description: |-
-            Maximum number of documents to collect for each shard. If a query reaches this
-            limit, OpenSearch terminates the query early. OpenSearch collects documents
-            before sorting. Defaults to 0, which does not terminate query execution early.
+            The maximum number of documents to collect for each shard. If a query reaches this
+            limit, OpenSearch stops the query early. OpenSearch collects documents
+            before sorting. Default is 0, which does not terminate query execution early.
           type: number
         stats:
           description: |-
-            Stats groups to associate with the search. Each group maintains a statistics
+            The statistics groups to associate with the search. Each group maintains a statistics
             aggregation for its associated searches. You can retrieve these stats using
-            the indices stats API.
+            the Index Stats API.
```

Comment:
Global: Can we change any instances of "stats" to "statistics"?

Reply:
Will do with the exception of this line, since it refers to a specific API (https://opensearch.org/docs/latest/api-reference/index-apis/stats/).